aten/src/ATen/TensorIterator.h

```diff
@@ -313,6 +313,10 @@ struct TORCH_API TensorIteratorBase : public impl::MetaBase {
   PtrVector get_data_ptrs(ArrayRef<char*> base, IntArrayRef counter) const;
   PtrVector get_base_ptrs() const;
 
+  // Helper functions for advanced stride manipulations (e.g. torch.flip)
+  void set_arg_strides(const int arg, IntArrayRef strides) { operands_[arg].stride_bytes = std::move(strides); }
```

Comment:
Probably makes sense to call these functions `_unsafe_set_arg_strides`, and `_unsafe_set_arg_data`, because no checks are made to validate the data